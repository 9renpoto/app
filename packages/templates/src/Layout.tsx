import { View } from 'react-native'
import cssta from 'cssta/native.macro'

export const Layout = cssta(View)`
  padding: 0px;

  @media (min-width: 600px) {
    padding: 20px;
  }
`
