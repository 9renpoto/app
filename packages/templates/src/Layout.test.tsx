import { shallow } from 'enzyme'
import { Layout } from './Layout'

describe('Layout', () => {
  it('render', () => {
    const wrapper = shallow(<Layout>hello</Layout>)
    expect(wrapper).toMatchSnapshot()
  })
})
