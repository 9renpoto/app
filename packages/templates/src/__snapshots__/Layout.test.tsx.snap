// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout render 1`] = `
<View
  style={
    Object {
      "paddingBottom": 20,
      "paddingLeft": 20,
      "paddingRight": 20,
      "paddingTop": 20,
    }
  }
>
  hello
</View>
`;
